src/tests/system/tests/test_passkey.py

```diff
@@ -0,0 +1,243 @@
+    :setup:
+        1. Add a LDAP, IPA, AD and Samba user with passkey_mapping.
+        2. Setup SSSD client with FIDO.
+    :steps:
+        1. Check authentication deny of the user when provided pin is incorrect.
+    :expectedresults:
+        1. User failed to authenticate.
+    :customerscenario: False
+    """
+    suffix = type(provider).__name__.lower()
+
+    if suffix == "ldap":
+        client.sssd.domain["local_auth_policy"] = "only"
+
+    with open(f"{testdatadir}/passkey-mapping.{suffix}") as f:
+        provider.user("user1").add().passkey_add(f.read().strip())
+
+    client.sssd.start()
+
+    assert not client.auth.su.passkey(
+        username="user1",
+        pin=67890,
+        device=f"{moduledatadir}/umockdev.device",
+        ioctl=f"{moduledatadir}/umockdev.ioctl",
+        script=f"{testdatadir}/umockdev.script.{suffix}",
+    )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.topology(KnownTopology.AD)
+@pytest.mark.topology(KnownTopology.Samba)
+def test_passkey__su_fail_mapping(client: Client, provider: GenericProvider, moduledatadir: str, testdatadir: str):
+    """
+    :title: Check authentication deny of user with LDAP, IPA, AD and Samba with incorrect mapping
+    :setup:
+        1. Add a LDAP, IPA, AD and Samba user with passkey_mapping.
+        2. Setup SSSD client with FIDO.
+    :steps:
+        1. Check authentication deny of the user when provided passkey mapping is incorrect.
+    :expectedresults:
+        1. User failed to authenticate.
+    :customerscenario: False
+    """
+    suffix = type(provider).__name__.lower()
+
+    if suffix == "ldap":
+        client.sssd.domain["local_auth_policy"] = "only"
+
+    with open(f"{testdatadir}/passkey-mapping.{suffix}") as f:
+        provider.user("user1").add().passkey_add(f.read().strip())
+
+    client.sssd.start()
+
+    assert not client.auth.su.passkey(
+        username="user1",
+        pin=123456,
+        device=f"{moduledatadir}/umockdev.device",
+        ioctl=f"{moduledatadir}/umockdev.ioctl",
+        script=f"{testdatadir}/umockdev.script.{suffix}",
+    )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.topology(KnownTopology.AD)
+@pytest.mark.topology(KnownTopology.Samba)
+def test_passkey__su_srv_not_resolvable(
+    client: Client, provider: GenericProvider, moduledatadir: str, testdatadir: str
+):
+    """
+    :title: Check authentication of a user with LDAP, IPA, AD and Samba when server is not resolvable
+    :setup:
+        1. Add a LDAP, IPA, AD and Samba user with passkey_mapping.
+        2. Setup SSSD client with FIDO.
+    :steps:
+        1. Check authentication of user before updating the server name in sssd.conf
+        2. Check authentication of user after updating the server name in sssd.conf
```

Comment:
You are missing a step in the middle to indicate that you are changing the server url in sssd.conf. This way you don't need to repeat `updating the server name in sssd.conf` in the steps.